_meeting-notes/2019-12-10.md

```diff
@@ -0,0 +1,150 @@
+Testground Weekly (45 mins)
+===
+
+###### tags: `IPFS` `Testing`
+
+:::info
+- **Host:** @daviddias
+- **Location:** https://protocol.zoom.us/j/181391682
+- **Date:** Dec 10, 2019 6:00 PM (UTC)
+- **Participants:**
+    - @raulk
+    - @daviddias
+    - @nonsense
+    - @hacdias
+    - @jimpick
+    - @stebalien
+- **Agenda**
+  ~~1. Sprint Update `00min`~~
+  2. Identify blockers / Answer Questions `30min`
+  3. This week's planning `15min`
+- **Reference:** 
+  - [Last week meeting minute 2019-12-03](https://github.com/ipfs/testground/blob/master/_meeting-notes/2019-12-03.md)
+  - [HOW_WE_WORK.md#weekly-sync](https://github.com/ipfs/testground/blob/master/docs/HOW_WE_WORK.md#weekly-sync)
+  - [KanBan Board](https://app.zenhub.com/workspaces/test-ground-5db6a5bf7ca61c00014e2961/board)
+  - [Testing Infra Design](https://github.com/ipfs/testground/blob/master/docs/SPEC.md)
+  - [Testing Inventory](https://github.com/ipfs/testground/blob/master/docs/test-inventory.md)
+  - [Testing Improvement Tracker](https://docs.google.com/spreadsheets/d/1xyqyGUF-oe3x9ln88YonVeOMWWdknik74lVgL_3dBY8/edit#gid=0)
+
+:::
+
+## :mega: Update (Done, Shipped, Made Progress)
+> To be reviewed Async
+
+- ==@raulk==
+- ==@daviddias==
+  - Planned Onboarding Hack Week https://docs.google.com/document/d/18P1EfbJ3T4SkafNur7u9uI7WqP8pReP340YXvy8EMAA/edit#
+  - Kicked off the Quarter Retrospective https://github.com/ipfs/testground/issues/220
+  - Unpacked Test Plan 4 for @jimpick https://github.com/ipfs/testground/issues/96#issuecomment-562877707
+  - Reviewed PRs
+  - Unblocked @hacdias
+  - Created issue templates https://github.com/ipfs/testground/pull/229
+  - Groomed the Kanban
+- ==@stebalien==
```

Comment:
Great update, @Stebalien. Thank you listing what you've achieved and what you haven't.